presto-kinesis/src/test/java/io/prestosql/plugin/kinesis/util/EmbeddedKinesisStream.java

```diff
@@ -95,8 +77,6 @@ public void delteStream(String streamName)
         DeleteStreamRequest deleteStreamRequest = new DeleteStreamRequest();
         deleteStreamRequest.setStreamName(streamName);
         amazonKinesisClient.deleteStream(deleteStreamRequest);
-        if (streamsCreated.contains(streamName)) {
-            streamsCreated.remove(streamName);
-        }
+        streamsCreated.remove(streamName);
```

Comment:
Can remove `streamsCreated` variable which is used only for `add` and `remove`? Also, this method has typo, "delteStream".